oneview/data_source_firmware_drivers.go

```diff
@@ -236,8 +233,10 @@ func dataSourceFirmwareDrivers() *schema.Resource {
 
 func dataSourceFirmwareDriversRead(d *schema.ResourceData, meta interface{}) error {
 	config := meta.(*Config)
-	id := d.Get("id").(string)
-	firmware, err := config.ovClient.GetFirmwareBaselineByNameandVersion(id)
+	name := d.Get("name").(string)
+	version := d.Get("version").(string)
+	id := name + version
```

Comment:
is there a possibility name comes with space? i mean id with space will not cause any issue?

Reply:
Name will can have space , but id is not used for for read.